### DISCUSSION TITLE
Dependabot should display line number from manifest

### DISCUSSION THREAD

**Discussion by sebastiantf** [community/community#33144] [Code Security] [answered]
It would be really helpful if Dependabot also showed the exact line number in the manifest (yarn.lock etc.), or even better, the full dependency graph (`yarn why --recursive`) of where the vulnerable dependency comes from

**Comment by bradify** [comment, marked as answer]
Hi @sebastiantf, thank you for the suggestion! The Dependabot team have been considering ways to make the information that is surfaced more valuable to developers through a more detailed source information. I like the simplicity of having the line number but that can get tricky for transitive dependencies in some package ecosystems. We do have the improvement for the UX of this experience on our backlog and are planning to address it!
